webapp/src/app/pages/components/knowledge-graph/concept-map/concept-map.component.css

```
@@ -508,3 +508,51 @@
   padding-right: 1rem;
   float: right;
 }
+
+::ng-deep .custom-close-btn {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)